docs/explore/airnode/implementation.md

```diff
@@ -14,57 +14,60 @@ tags:
-which requires the protocol to be designed to fit this scheme.
+need to be on call 24/7 in case of incidents. Since this is not a realistic
+requirement for first-party oracles, an oracle node designed for these oracles
+has to be stateless.
+
+However, there's another way to approach state keeping for oracle nodes: since
+the node on a given blockchain (such as Ethereum) used by the oracle node
+already keeps the state on its behalf, there is no need for the node itself to
+keep its state. Duplicating the state would create twice as many points of
+failure (and it would be enough for one of them to fail to cause total failure).
+Therefore, the oracle node should depend on the blockchain node to keep its
+state, which should be reflected in the way in which the oracle node's protocol
+is designed
 
 ### Non-idempotent operations
 
 An API operation is idempotent if calling it multiple times has the same effect
-as calling it once. For example, using a GET operation of an exchange API to get
-the ETH/USD price data is typically an idempotent operation. Calling it once or
-more will not make any difference at the API server-side. In contrast, using a
-POST operation of a remittance service provider API to send $100 to another
-party would be a non-idempotent operation. Each call would send an additional
-$100, and thus using the operation multiple times would have a different effect
-than using it once.
-
-The oracle node being stateless means that it would not be able to "remember" if
-it has made an API call associated with a request, and may repeat it under
-certain conditions. This is not an issue at the moment, because presently,
-oracles are only used for idempotent operations. The aim is for Airnode to
-support non-idempotent operations as well. There is research into alternative
-methods to achieve this while protecting the resiliency that statelessness
-provides.
+as calling it once. For example, using a GET operation of an exchange's API to
+get the exchange rate between ETH and USD is typically an idempotent operation.
+It will not make any difference at the API server-side if we call it once or
+several times.
+
+In contrast, using a POST operation of a remittance service provider API to send
+$100 to another party would be a non-idempotent operation. Each call would send
+an additional $100, which means that using the operation multiple times would
+have a different effect than using it once.
+
+Stateless oracle nodes cannot "remember" if they already made an API call
+associated with a given request, and, under certain conditions, they may repeat
+it. At present, this is not an issue since oracles are only used for performing
+idempotent operations. In the future, however, Airnode intends to support
+non-idempotent operations as well. We are currently researching alternative
+methods of achieving this while protecting the resiliency provided
+bystatelessness.
```

Comment:
The `bystatelessness` word feels too complex.

Reply:
@martinkolenic : Do you want to do something with `bystatelessness`?

Reply:
This should be two words `by statelessness`

Reply:
@dcroote  - @martinkolenic : I will make the change suggested by Derek.

Reply:
Hi, @dcroote and @wkande sorry for the late reply, I was, indeed, off due to illness. The extra long word was a mistake on my part, no idea why it got past the spellchecker. Thanks for sorting that out. I'll review the rest of the Explore section and then I'll reach out to you, Warren, to discuss any other documents I could work on while I'm still on the project.